src/logo-system/logo-elements/index.md

```diff
@@ -50,14 +50,14 @@ The size of the crown can be adjusted depending on context. For example, when be
 Scaling should follow this rule:
 Wordmark dot = 2× crown dot
 
-![TODO](./propotional-scaling-desktop.svg)
+![Dots from the crown are used to show the correct scale between the crown and wordmark.](./propotional-scaling-desktop.svg)
 
 #### Enlarged crown size
 
 Scaling should follow this rule:
 Wordmark dot = 1× crown dot
 
-![TODO](./propotional-scaling-mobile.svg)
+![Dots from the crown are used to show the correct scale between the crown and wordmark for use in mobile.](./propotional-scaling-mobile.svg)
```

Comment:
(Optional) Same question about clarity as above. (It shows the correct size of the dot between the "GOV" and "UK".)